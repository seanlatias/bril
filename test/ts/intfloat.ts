var value = 9n
var result = value / 20n
console.log(result)
console.log(9 / 20)
// Floats are unsupported at the moment