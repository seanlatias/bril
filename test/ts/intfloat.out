main {
  v0: int = const 9;
  value: int = id v0;
  v1: int = id value;
  v2: int = const 20;
  v3: int = div v1 v2;
  result: int = id v3;
  v4: int = id result;
  print v4;
  v5: int = const 0;
  v6: double = const 9;
  v7: double = const 20;
  v8: double = fdiv v6 v7;
  print v8;
  v9: int = const 0;
}
